frontend/src/components/project/Buttons/ContactCreatorButton.js

```diff
@@ -33,16 +32,17 @@ const useStyles = makeStyles({
   slideInTitle: {
     fontWeight: "bold",
   },
-  buttonWithCollapseContainer: {
+  collapsableContainer: {
     position: "absolute",
     bottom: 0,
     right: 0,
   },
+  applyCustomCardWidth: {
+    width: "100%",
```

Comment:
If there is a reason why you need these width properties, you can ofc keep them! :-) As I was doing the review, it seemed that some of them were redundant but it's fully ok to have them. Thank you for checking it. :-)